aws-storage-s3/src/test/java/com/amplifyframework/storage/s3/AWSS3StoragePluginTest.kt

```diff
@@ -144,7 +143,7 @@ class AWSS3StoragePluginTest {
 
         plugin.configure(data, mockk())
         val bucket = StorageBucket.fromOutputs("myBucket")
-        shouldThrow<InvalidStorageBucketException> {
+        shouldThrow<StorageException> {
             plugin.getStorageService(bucket)
         }
```

Comment:
Can check that the cause matches expectation

```suggestion
        val exception = shouldThrow<StorageException> {
            plugin.getStorageService(bucket)
        }
        exception.cause.shouldBeInstanceOf<InvalidStorageBucketException>()
```

Reply:
very cool, unit test updated per suggestion.